lib/cdo/shared_constants/pd/teacher1819_application_constants.rb

```diff
@@ -135,11 +137,16 @@ def self.clean_multiline(string)
   }
 
   VALID_SCORES = {
-    able_to_attend_single: ['Yes', 'No'],
-    committed: ['Yes', 'No'],
+    regional_partner_name: YES_NO,
+    # Need to include whether the participant is new
```

Comment:
Is this change coming in a separate PR/being tracked somewhere?

Reply:
Yeah - Andrew's PR